addons/mail/tests/test_res_users_settings.py

```diff
@@ -1,14 +1,34 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from odoo.addons.mail.tests.common import MailCommon
-from odoo.tests.common import tagged, users
+from odoo.tests.common import tagged, users, TransactionCase
+
+
+class TestUSerCreationSettings(TransactionCase):
+
+    def test_create_portal_user(self):
+        portal_group = self.env.ref('base.group_portal')
+        user = self.env.user.create({
+            'name': 'A portal user',
+            'login': 'portal_test',
+            'groups_id': [(6, 0, [portal_group.id])],
+        })
+        self.assertFalse(user.res_users_settings_ids, 'Portal users should not have settings by default')
```

Comment:
maybe remove comment, the assert is clean and the comment is making it harder to read than it should